### DISCUSSION TITLE
<title>Question about push a branch

### DISCUSSION THREAD

**Discussion by Arnei-Dev** [community/community#42173] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi guys, today I tried to push my branch and set the remote as upstream, therefore I just received the following command from my terminal "use: git push --set-upstream arigin my branch". I'm a beginner in a programming language,  but I wonder if "arigin" is correct? instead origin. 
"

**Comment by byrneh** [comment]
Yes origin

**Comment by getlost01** [comment]
No, "arigin" is not correct. "origin" is the correct name for the default remote repository in Git. The command you received from your terminal, "`git push --set-upstream arigin my branch`", is incorrect and will not work.

To set the upstream repository for your branch and push your changes to it, you should use the following command:

`git push --set-upstream origin my-branch`

Replace "my-branch" with the name of your branch. This will push your changes to the "origin" remote repository and set it as the upstream repository for your branch.

It's important to make sure that you use the correct commands when working with Git, as mistakes can lead to problems with your repository and its history.

**Comment by DSDmark** [comment]
Hi @Arnei-Dev,

The correct command should be like this.

> `git push --set-upstream origin my_branch`

`Origin` is the name of the remote repository, and `my_branch` is the name of your local branch that you want to push to the remote repository. `arigin` is not a correct command, and it is likely a typo.

**Comment by Arnei-Dev** [comment]
Thanks man.

**Reply by Musthuakqha** [reply]
🙂
